### ISSUE TITLE
using <spec path>/index.jsx doesn't work

### ISSUE THREAD

**Issue by mik01aj** [sourcejs/sourcejs-react-docgen#1]
It's a bug in this line: https://github.com/sourcejs/sourcejs-react-docgen/blob/82f8bc15463c264a8faa65aecf454fa6b75919d0/core/middleware/index.js#L41

CC @operatino

**Comment by robhrt7** [comment]
Thanks for the report, I'll check it out. Was doing thus plugin in a rush for a demo.

**Comment by mik01aj** [comment]
Even after moving the file, the plugin still doesn't work:

![image](https://cloud.githubusercontent.com/assets/3027415/10824231/e87ba1b2-7e5f-11e5-844a-760387dd159e.png)

This is the `docgen` output for my component:

```
{
  "description": "",
  "displayName": "Avatar",
  "props": {
    "user": {
      "type": {
        "name": "shape",
        "value": {
          "name": {
            "name": "string",
            "required": true
          }
        }
      },
      "required": false,
      "description": ""
    },
    "size": {
      "type": {
        "name": "number"
      },
      "required": false,
      "description": ""
    },
    "withTooltip": {
      "type": {
        "name": "bool"
      },
      "required": false,
      "description": ""
    },
    "style": {
      "type": {
        "name": "object"
      },
      "required": false,
      "description": ""
    }
  }
}
```

**Comment by robhrt7** [comment]
Fixed in 0.2.0
